### ISSUE TITLE
NTR: MRI: compression of seventh cranial nerve by blood vessel (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8910]
New term request MRI: compression of seventh cranial nerve by blood vessel (kalia_100601032127): NVCC; neurovascular cross-compression; facial; 7th; VII  UMLS: 0521670

**Comment by pnrobinson** [comment]
@MickeySegal
the HPO would not use the "by blood vessel", but here is something close

Adding new term: 
- ID: HP:6000411
- label: Facial nerve compression
- parents: [Cranial nerve compression (HP:0001293)](https://hpo.jax.org/app/browse/term/HP:0001293)
- pmid: 
- def: Impingement of another object such as a blood vessel upon the facial nerve, causing it to lose volume in the compressed region.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8910
- synonyms: Compression of seventh cranial nerve(NONE)
